tests/FlowRiemannShockTube/input

```diff
@@ -5,7 +5,7 @@
 #@ [roe-fix]
 #@ exe=hydro
 #@ dim=2
-#@ roefix=1
```

Comment:
@etsandall  This statement actually had no effect, because it was not specified with `args = `. So the reason why it ran faster before was that it wasn't running the entropy fix at all  (and therefore was not producing all the messages which slowed it down).